packages/template-chakra-storefront/src/pages/product-detail/index.test.js

```diff
@@ -51,36 +40,49 @@ jest.mock('../../hooks/use-datacloud', () => ({
 
 const MockedComponent = () => {
     return (
-        <Switch>
-            <Route
-                path="/uk/en-GB/product/:productId"
-                render={(props) => <ProductDetail {...props} />}
-            />
-        </Switch>
+        <>
+            <Switch>
+                <Route
+                    path="/uk/en-GB/product/:productId"
+                    render={(props) => <ProductDetail {...props} />}
+                />
+            </Switch>
+            <Toaster toaster={toaster} />
```

Comment:
Was this Toaster removed at one point?

Reply:
Not really, The way Toast is used in Chakra v3 differ to V2. 
In V2, the Toast is sorta baked into the DOM, and you only need to call `useToast` to render it. See [here](https://v2.chakra-ui.com/docs/components/toast)

In V3, it broke it down into two sections, one is Toaster UI, and another is useToast hook. You need both to get the toast working. See [here](https://www.chakra-ui.com/docs/components/toast#usage)